.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+//https://stackoverflow.com/questions/42495655/how-to-debug-angular-with-vscode

```

Comment:
I don't use VS Code but having comments ("//") in the file is invalid JSON in every other context. But if it works, it works :)

Reply:
I will remove them that way if anything else uses those files it would work. PS you would try VSCode. It works great on Linux too. It was the best IDE that I have discovered.

Reply:
I tried VSCode. It's totally fine, but I prefer WebStorm.

Reply:
Strange, Google uses VSCode exclusively and everybody loves it. not only that they contribute to the project. :)

Reply:
WebStorm has some powerful features VSCode doesn't, especially when it comes to refactoring. In general the way it analyses the code seems quite a bit smarter than any other IDE or editor I've worked with so far. But VSCode is not a bad choice either. There are good reasons to use it (e.g. speed, customization options).

Also it takes a lot of time to get used to a new IDE. It would take month  if not years for me to get as fast with VSCode as I'm with WebStorm.